### PR TITLE
[INFRA/IIGO] Broadcasting sanction thresholds

### PR DESCRIPTION
**Pull request by Neelesh99**
# Summary

Since we are now allowing the Judge to load custom sanction thresholds, it's important that islands know about them.

## Test Plan

Tests added for broadcast generation.

### CODE REVIEW

**Comment by QFSW** [comment]
# ⚠️Warning! This diff contains changes in protected folders

**Comment by preetl** [comment]
SO PROUD OF YOU FOR MAKING A LIL PR!!!

File: internal/common/shared/iigocommunications.go [hunk 1]
```diff
@@ -71,6 +71,8 @@ const (
 	SanctionAmount
 	RoleMonitored
 	MonitoringResult
+	IIGOSanctionTier
+	IIGOSanctionScore
```
Comment (lhl2617) [inline internal/common/shared/iigocommunications.go:75]: did you update the String() function

Comment (Neelesh99) [reply internal/common/shared/iigocommunications.go:75]: Crap, just realised this Enum hasn't been moved over to that model. Let me add that.

**Review by lhl2617 — APPROVED** [review]
image

thanks for the smol pr

pre-approving, feel free to merge after fixing String() for enum

File: internal/common/shared/iigocommunications.go (hunk 1 above)
Comment (Neelesh99) [reply internal/common/shared/iigocommunications.go:75]: Nvm I was confused, added to the .String() function

**Pull request merged**